tools/RAiDER/cli/raiderDelay.yaml

```diff
@@ -0,0 +1,60 @@
+# vim: set filetype=yaml:
+##------------------------ raiderDelay.cfg ------------------------##
+########## 1. Date and time
+## Dates can be a single date, two dates that define a set of dates every day, or two dates with an interval
+## Standard formats are accepted. 
+date_start:   # first date to download. Specified as YYYYMMDD
+date_end:   # last date to download, auto for None (single date)
+date_step:   # date interval to download; e.g. 12 for every 12 days. Auto for none (downloads every day)
+date_list:   # Pass in a list of dates directly to process
+time:   # time to download in UTC time. Specified as HH:MM:SS or HH:MM
+
+########## 2. Weather model
+## Currently implemented weather models include: 
+## ERA-5 (ECMWF), ERA-5T (ECMWF), HRES (ECMWF), ERA-I (ECMWF), NCMR (ISRO), HRRR (NOAA), GMAO (NASA), MERRA-2 (NASA)
```

Comment:
I think here we should clarify when not providing what the default is

Reply:
@dbekaert yep I'm not sure best either, some fields are required and some have defaults. We can link back to [this page](https://github.com/dbekaert/RAiDER/blob/dev/WeatherModels.md) for documentation.